web_programming/open_google_results.py

```diff
@@ -13,26 +22,19 @@
 
     url = f"https://www.google.com/search?q={query}&num=100"
 
-    res = requests.get(
+    res = httpx.get(
         url,
         headers={"User-Agent": str(UserAgent().random)},
         timeout=10,
     )
+    print(BeautifulSoup(res.text, "html.parser"))
```

Comment:
Extra `print`. Maybe this was added for debug purpose?

Reply:
```suggestion
```